maven-scm-test/src/main/java/org/apache/maven/scm/tck/command/changelog/ChangeLogCommandTckTest.java

```diff
@@ -94,7 +94,6 @@ public void testChangeLogCommand()
         ChangeSet changeset = thirdResult.getChangeLog().getChangeSets().get( 0 );
         assertTrue( changeset.getDate().after( timeBeforeSecond ) );
 
-
-        assertEquals( COMMIT_MSG, changeset.getComment() );
+        assertTrue( changeset.getComment().startsWith( COMMIT_MSG ) ); // Sometimes there is a newline appended
```

Comment:
Under which circumstances did this occur to you?

Reply:
One of the SCMs returned it with a newline. Perhaps this should be fixed differently.

Reply:
Which? I would rather strip trailing LS and then go through `assertEquals()`

Reply:
Turns out that the HG code ... appends a newline. Fixed